docs/ref/utils.txt

```diff
@@ -351,11 +351,17 @@ SyndicationFeed
         All parameters should be Unicode objects, except ``categories``, which
         should be a sequence of Unicode objects.
 
-    .. method:: add_item(title, link, description, author_email=None, author_name=None, author_link=None, pubdate=None, comments=None, unique_id=None, enclosure=None, categories=(), item_copyright=None, ttl=None, updateddate=None, **kwargs)
+    .. method:: add_item(title, link, description, author_email=None, author_name=None, author_link=None, pubdate=None, comments=None, unique_id=None, enclosures=None, categories=(), item_copyright=None, ttl=None, updateddate=None, **kwargs)
```

Comment:
I'd keep `enclosure=None` since it's still part of the API. Also, `enclosures=None` needs to be moved to next of `updateddate=None`.